### PR TITLE
#111 - update readme

### PR DESCRIPTION
**Pull request by aw-west-defra**
#111

### CODE REVIEW

**Review by obi-defra — APPROVED** [review]
Made a couple of suggested edits to the read me, just to make text a bit clearer.

File: README.md [hunk 1]
```diff
@@ -1,97 +1,68 @@
 # elmo-geo
+This repository is an companion to [ELMO][elmo]. It has been separated to work on [DASH][dash] for large scale geospatial analysis.  Primarily, the outputs from elmo-geo inform the missing [eligibility criteria][elig] for ELMO.
 
-## Setup
+| Status | Project | Description |
+| ------ | ------- | ----------- |
+| :gear: | Ingestion | [readme](notebooks/ingestion/readme.md)
```
Comment (burrowsej) [inline README.md:6]: This link doesn't work for me

**Review by burrowsej — APPROVED** [review]
Think this is a good improvement thanks, no real comments.
I think better ways of working might be explored in the future, e.g. using VSCode in AVD with Databricks extension but takes time to explore these things...

File: README.md (hunk 1 above)
Comment (aw-west-defra) [reply README.md:6]: It works in main, but not branch.

**Pull request merged**